src/System.Memory/src/System/Buffers/Text/StandardFormat.cs

```diff
@@ -0,0 +1,123 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Buffers.Text
+{
+    /// <summary>
+    /// Represents a standard formatting string without using an actual String. A StandardFormat consists of a character (such as 'G', 'D' or 'X')
+    /// and an optional precision ranging from 0..99, or the special value NoPrecision.
+    /// </summary>
+    public struct StandardFormat : IEquatable<StandardFormat>
```

Comment:
Applicable structs were recently marked readonly throughout the repo (#24997). Looks like this could be readonly as well.

Reply:
Fixed.